examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,825 @@
+{
```

Comment:
<p>May need to say that <code>h</code>  still has the size of the full domain, not the size of the condition.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
How do you conceive here the size of condition?

Reply:
the size of the domain that satisfies the condition

Reply:
Sure, fixed.